src/math/Matrix3.d.ts

```diff
@@ -1,6 +1,12 @@
 import { Matrix4 } from './Matrix4';
 import { Vector3 } from './Vector3';
 
+type Matrix3tuple = [
+	number, number, number,
+	number, number, number,
+	number, number, number,
+];
```

Comment:
Consider using shorter gl-matrix style names here, like `mat4`: http://glmatrix.net/docs/.

Reply:
That's cool, yes you posted those definitions to my bug report.  That was very helpful, I used that as my starting point.

I'm happy to go with whatever the prefered naming conventions are around here, but first I should explain why according to my own taste I extended those names, so that `mat4` became `Matrix4tuple`

- To disambiguate type-names from variable-names I uppercase the first letter of a type, and lowercase the first letter of a variable.  [I think this is fairly standard](https://basarat.gitbook.io/typescript/styleguide#type).
- I added the word `tuple` to make it clear that this type is a tuple, and thus help to disambiguate it from the Matrix4 type. 
- `mat` became `matrix` to make it clear to those not familiar with mathematical shorthand that this isn't something to wipe your feet on (or a material or mesh with a mat-like quality)

Having clarified that, I'll naturally be more than happy to restore the original type names if those are prefered.

Reply:
The names used by gl-matrix are also used by the WebGL API itself, e.g. https://thebookofshaders.com/glossary/?search=mat4, so to me it's more of a WebGL idiom here. But if we did want a more explicit name I'd suggest `Matrix4Tuple`, `ArrayMatrix4`, or `Matrix4Array`. Since we shouldn't need to export this type definition, maybe it doesn't make too much difference.

Reply:
That's cool!  Someone will have to decide about short names vs. more explicit names, though they may want to let the dust settle here first.

But for explicit names, I will at minimum need to upper-case the 'T' in 'tuple' to convert my suggestion of `Matrix4tuple` into your `Matrix4Tuple`, thus correcting my SnakeCase.

Reply:
Ok – Yeah I'm fine with any of the names here really, since it's not exported. If others have feedback feel free to comment, but I don't mean to block the PR on a change to the names.

Reply:
Thanks :-)  SnakeCase fixed as promised.